index.html

```diff
@@ -0,0 +1,203 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Daiane Alves</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="./css/flexslider.css" />
+    <link rel="stylesheet" type="text/css" media="screen" href="estilo.css" />
+    <link rel="icon" href="./images/icon.ico" type="image" sizes="16x16">
+
+
+</head>
+
+<body>
+    <header id="menu">
+        <figure>
+            <img src="images/logo.png" height="80">
+        </figure>
+        <nav id="menu-horizontal" style="float:right">
+            <ul>
+                <li style="display:inline"><a href="#">HOME</a></li>
```

Comment:
Colocar as referências para as respectivas seções
```suggestion
                <li style="display:inline"><a href="#banner">HOME</a></li>
```